ert3/console/_errors.py

```diff
@@ -0,0 +1,4 @@
+def report_validation_errors(message: str, source=None) -> None:
```

Comment:
Just a thought that might be a bad idea... Could this be the `__str__` method of the exception instead? Such that validation errors do not have be treated separately from `ErtError`'s in the main method of the CLI?

Reply:
Yes, if we want to implicitly handle ValidationError, which maybe we should not (see previous comment). But then we end up with a generic function that would dump any error just by printing its string. That is fine for pydantic which does very nice error formatting. But for other errors that may not be so, and it may be better to force the developer to think about what they are going to print first.

Reply:
As mentioned above, I think we should only catch ERT specific errors here. And then this question really boils down to whether it should be the exception or the cli's responsibility to display the error in a text-friendly manner.

I'm fine with either, as both has their merits... Do as you see best 👍🏻

Reply:
When I think about it, this function is really  meant for reporting ConfigValidationError exceptions only. So I will pass the exception, and make sure the the typing annotation specifies ConfigValidationError.